manual/src/html_processing/scss/manual.scss

```diff
@@ -185,6 +185,9 @@ h3 code{
 h4{
     font-size:1.12em
 }
+h2, h3, h4 {
+	font-weight: 500;
+}
```

Comment:
This change is non-essential and purely subjective. I think it made it easier to skim for subsection headings and adds some visual variation to the whole thing, gives less of a "wall of text" feeling.

Reply:
I see your point and that seems potential nicer to me. It might be worthwhile to get a third advice (maybe on a larger text fragment to see how multiple titles mesh with the rest of the text?).

Reply:
Actually, this change is not without precedent, as both the old htmlman and LaTeX use a slightly higher font weight for those headings.